tests/ssl-opt.sh

```diff
@@ -752,6 +752,23 @@ run_test() {
     rm -f $SRV_OUT $CLI_OUT $PXY_OUT
 }
 
+run_test_psa() {
+    requires_config_enabled MBEDTLS_USE_PSA_CRYPTO
+    run_test    "Default, PSA ($1)" \
```

Comment:
Please remove "Default" from the test description - this in only for test whose purpose is explicitly to check the defaults (ie, nothing forced, except maybe DTLS), so it's inappropriate here. Please use something more descriptive, like "PSA used for record en/decryption".

(There are already two tests that include "Default" in there name while they shouldn't and that already annoys me. I want to be able to run `tests/ssl-opt.sh -f Default` for a very quick test that (D)TLS work in the default settings and nothing else. If I wanted tests for specific things in addition to that, I'd use something else than `-f Default`.)

Reply:
Fixing.